addons/web_editor/controllers/main.py

```diff
@@ -344,8 +345,30 @@ def get_image_info(self, src=''):
         """This route is used to determine the original of an attachment so that
         it can be used as a base to modify it again (crop/optimization/filters).
         """
+        def _get_attachment_by_url(src):
+            """Find attachment by url. There can be multiple matches because of
+            default snippet images referencing the same image in /static/, so we
+            limit to 1.
```

Comment:
Of course there should be a `limit=1` but I don't understand your explained reason here.